tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_progress_bar.py

```diff
@@ -21,8 +23,11 @@ def test_run():
     for epoch in range(max_epochs):
         for _ in range(iters_per_epoch):
             with manager.run_iteration():
+                time.sleep(0.1)
                 if manager.iteration < 2:
                     continue
-                status = '{} iter, {} epoch / {} epochs'.format(
-                    manager.iteration, epoch, max_epochs)
-                assert status in out.getvalue()
+                status = out.getvalue()
+                assert '{} iter, {} epoch / {} epochs'.format(
+                    manager.iteration, epoch, max_epochs) in status
+                iters_per_sec = re.findall(r'[0-9]+\.[0-9]* iters/sec', status)[-1]
+                assert 8 <= float(iters_per_sec[:-10]) <= 12
```

Comment:
Could you write this with `re.search` with grouping for better readablity?

Reply:
Fixed the regexp pattern to avoid dirty trimming.